### PR TITLE
[instant] Don't throw error if can't find icon for token

### PR DESCRIPTION
**Pull request by steveklebanoff**
## Description

Don't throw an error if we can't find an icon for a token

## Testing instructions

Use custom erc20 integration code here: https://gist.github.com/steveklebanoff/b87eef363f6df06a144426120eb731d9

## Checklist:

*   [ ] Prefix PR title with `[WIP]` if necessary.
*   [ ] Add tests to cover changes as needed.
*   [ ] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/instant/src/components/erc20_token_selector.tsx [hunk 1]
```diff
@@ -123,10 +122,20 @@ interface TokenSelectorRowIconProps {
     token: ERC20Asset;
 }
 
+const getTokenIcon = (symbol: string): React.StatelessComponent | undefined => {
+    try {
+        return require(`../assets/icons/${symbol}.svg`) as React.StatelessComponent;
```
Comment (BMillman19) [inline packages/instant/src/components/erc20_token_selector.tsx]: hmm dunno if we have to go as far as casting this as a stateless component

Comment (steveklebanoff) [reply packages/instant/src/components/erc20_token_selector.tsx]: i think it's unlikely that a required svg would have state, and I like the typesafety of the null checking, but I can change back to `any` if you feel strongly

Comment (fragosti) [reply packages/instant/src/components/erc20_token_selector.tsx]: I've always thought there should be a separate file where we iterate over all the token symbols in our data and require the icon. This also ensures that we (the developers) can't include a new token without an svg.

Comment (steveklebanoff) [reply packages/instant/src/components/erc20_token_selector.tsx]: I think this would be a good CI task

**Pull request merged**